requirements-dev.txt

```diff
@@ -2,6 +2,7 @@
 
 # For running doctests (during "python setup.py test")
 Sphinx >= 1.4.5
+sphinx_bootstrap_theme
```

Comment:
Hum.... that makes sense.
But, is sphinx_bootstrap_theme required here? If so, why not sphinx_rtd_theme too (it is a requirement for bootstrap_theme)?